core/pom.xml

```diff
@@ -26,4 +26,45 @@
 
     <artifactId>core</artifactId>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${compiler.plugin.verson}</version>
+                <executions>
+                    <execution>
+                        <id>compile-jdk8</id>
```

Comment:
The default `default-compile` execution already creates java 8 code. This does the same, causing the java 8 code to be compiled twice.